proto/frequenz/api/reporting/v1/reporting.proto

```diff
@@ -312,6 +327,9 @@ message ReceiveMicrogridComponentsDataStreamRequest {
     // Optional time-based filter criteria.
     // If omitted, data will start streaming from the timestamp that the request was received.
     TimeFilter time_filter = 3;
+
+    // List of metric source filters
+    repeated MetricSourceOptions metric_sources = 4;
```

Comment:
We should add this to the non-streaming RPC as well.

Reply:
That one is being deprecated though. Do we really want to keep maintaining it if nobody is going to use it?

Reply:
No, makes sense to add it there then.